redbot/core/app_commands/errors.py

```diff
@@ -0,0 +1,10 @@
+"""Errors module for the app_commands package."""
+from discord import app_commands
+
+
+class UserFeedbackCheckFailure(app_commands.CheckFailure):
+    """A generic version of CheckFailure."""
```

Comment:
The docstring from `commands.CheckFailure` makes more sense than this one:
```suggestion
    """A version of CheckFailure which isn't suppressed."""
```

Reply:
i was gonna use that as well, but unlike `Red.on_command_error` `app_commands.CheckFailure` isn't really suppressed on `RedTree.on_error` https://github.com/Cog-Creators/Red-DiscordBot/blob/efc99c5dceda82537f75c24f734d4fb4dd4a8e11/redbot/core/tree.py#L336

Reply:
would there be something more appropriate in this case?

Reply:
I see... Maybe something like this?
```suggestion
    """A version of CheckFailure responding with a custom error message."""
```

Reply:
I suppose this would be a better one for now.